src/test/java/io/nats/client/ConnectTests.java

```diff
@@ -397,24 +390,20 @@ public void testAsyncConnectionWithReconnect() throws IOException, InterruptedEx
             Nats.connectAsynchronously(options, true);
 
             // No server at this point, let it fail and try to start over
-            try {
-                Thread.sleep(1000);
-            } catch (Exception exp) {
-
-            }
+            try { Thread.sleep(5000); } catch (Exception exp) { /* ignored */ }

```

Comment:
Increased the time given to fail. The test expects it to fail, that's the point of the test, not how long it takes to fail.